web/app/scripts/src/providers/project-config.es.js

```diff
@@ -0,0 +1,22 @@
+const ProjectConfig = function () {
+    let settings = {
+        apiRoot: 'localhost:8000'
+    };
+
+    return {
+        $get: function () {
+            return {
+                getApiRoot: () => settings.apiRoot
```

Comment:
That's what I think is too much work to be worth it: every time you need to add a new property, say `powerLevel: 9001`, you have to add `getPowerLevel: () => settings.powerLevel`